www/source/blog/2018-11-27-0.69.0-release.html.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Habitat 0.69.0 Released
+date: <2018-11-27>
+author: Scott Macfarlane
+tags: release notes
+category: product
+classes: body-article
+---
+
+Habitat 0.69.0 Release Notes
+
+We are happy to announce the release of Habitat 0.69.0. We have a number of new features as well
+as bug fixes, so please read on for all the details. If you just want the binaries, head on over to
+[Install Habitat](https://www.habitat.sh/docs/using-habitat/#install-habitat).
+
+Thanks again for using Habitat!
+
+Highlights:
+#### New Features & Enhancements
+- Add Windows-friendly behavior to `hab pkg env` [#5813](https://github.com/habitat-sh/habitat/pull/5813) ([mwrock](https://github.com/mwrock))
+- Add -t to support installing alternate PackageTarget cli  [#5770](https://github.com/habitat-sh/habitat/pull/5770) ([smacfarlane](https://github.com/smacfarlane))
+
+#### Behavioral Changes
+- Add target to Docker image names for studio [#5805](https://github.com/habitat-sh/habitat/pull/5805) ([smacfarlane](https://github.com/smacfarlane))
+
+#### Bug Fixes
+- Restart the election if the leader is dead [#5859](https://github.com/habitat-sh/habitat/pull/5859) ([baumanj](https://github.com/baumanj))
+- Exclude services from things you can uninstall [#5835](https://github.com/habitat-sh/habitat/pull/5835) ([jamesc](https://github.com/jamesc))
+- Miscellaneous code cleanup [#5857](https://github.com/habitat-sh/habitat/pull/5857) ([baumanj](https://github.com/baumanj))
+- Clean up a variety of leaks [#5828](https://github.com/habitat-sh/habitat/pull/5828) ([raskchanky](https://github.com/raskchanky))
+
+#### Merged Pull Requests
```

Comment:
Most of these should be categorized; they were just missing the appropriate `X-` labels. I'll leave the ones @christophermaier and @mpeck did to their discretion, but the one from rsertelon should be a "Behavioral Change" and the one from jeremymv2 should be a "Bug Fix".

Reply:
Done, and thank you!